library/drupal

```diff
@@ -4,35 +4,35 @@ Maintainers: Tianon Gravi <admwiggin@gmail.com> (@tianon),
+Tags: 11.1.0-php8.3-fpm-bookworm, 11.1-php8.3-fpm-bookworm, 11.1.0-php8.3-fpm, 11.1-php8.3-fpm, 11.1.0-fpm-bookworm, 11.1-fpm-bookworm, 11.1.0-fpm, 11.1-fpm
 Architectures: amd64, arm32v7, arm64v8, i386, ppc64le, s390x
-GitCommit: 561a5f6557e485370c336becc01e9bdb7ef582ac
-Directory: 11.1-rc/php8.3/fpm-bookworm
+GitCommit: dfb409af0becc76411b367ff2107209d7a1f3611
+Directory: 11.1/php8.3/fpm-bookworm
 
-Tags: 11.1.0-rc1-php8.3-apache-bullseye, 11.1-rc-php8.3-apache-bullseye, 11.1.0-rc1-apache-bullseye, 11.1-rc-apache-bullseye
+Tags: 11.1.0-php8.3-apache-bullseye, 11.1-php8.3-apache-bullseye, 11.1.0-apache-bullseye, 11.1-apache-bullseye
 Architectures: amd64, arm32v7, arm64v8, i386
-GitCommit: 561a5f6557e485370c336becc01e9bdb7ef582ac
-Directory: 11.1-rc/php8.3/apache-bullseye
+GitCommit: dfb409af0becc76411b367ff2107209d7a1f3611
+Directory: 11.1/php8.3/apache-bullseye
 
-Tags: 11.1.0-rc1-php8.3-fpm-bullseye, 11.1-rc-php8.3-fpm-bullseye, 11.1.0-rc1-fpm-bullseye, 11.1-rc-fpm-bullseye
+Tags: 11.1.0-php8.3-fpm-bullseye, 11.1-php8.3-fpm-bullseye, 11.1.0-fpm-bullseye, 11.1-fpm-bullseye
 Architectures: amd64, arm32v7, arm64v8, i386
-GitCommit: 561a5f6557e485370c336becc01e9bdb7ef582ac
-Directory: 11.1-rc/php8.3/fpm-bullseye
+GitCommit: dfb409af0becc76411b367ff2107209d7a1f3611
+Directory: 11.1/php8.3/fpm-bullseye
 
-Tags: 11.1.0-rc1-php8.3-fpm-alpine3.21, 11.1-rc-php8.3-fpm-alpine3.21, 11.1.0-rc1-php8.3-fpm-alpine, 11.1-rc-php8.3-fpm-alpine, 11.1.0-rc1-fpm-alpine3.21, 11.1-rc-fpm-alpine3.21, 11.1.0-rc1-fpm-alpine, 11.1-rc-fpm-alpine
+Tags: 11.1.0-php8.3-fpm-alpine3.21, 11.1-php8.3-fpm-alpine3.21, 11.1.0-php8.3-fpm-alpine, 11.1-php8.3-fpm-alpine, 11.1.0-fpm-alpine3.21, 11.1-fpm-alpine3.21, 11.1.0-fpm-alpine, 11.1-fpm-alpine
 Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, riscv64, s390x
-GitCommit: 561a5f6557e485370c336becc01e9bdb7ef582ac
-Directory: 11.1-rc/php8.3/fpm-alpine3.21
+GitCommit: dfb409af0becc76411b367ff2107209d7a1f3611
+Directory: 11.1/php8.3/fpm-alpine3.21
 
-Tags: 11.1.0-rc1-php8.3-fpm-alpine3.20, 11.1-rc-php8.3-fpm-alpine3.20, 11.1.0-rc1-fpm-alpine3.20, 11.1-rc-fpm-alpine3.20
+Tags: 11.1.0-php8.3-fpm-alpine3.20, 11.1-php8.3-fpm-alpine3.20, 11.1.0-fpm-alpine3.20, 11.1-fpm-alpine3.20
 Architectures: amd64, arm32v6, arm32v7, arm64v8, i386, ppc64le, riscv64, s390x
-GitCommit: 561a5f6557e485370c336becc01e9bdb7ef582ac
-Directory: 11.1-rc/php8.3/fpm-alpine3.20
+GitCommit: dfb409af0becc76411b367ff2107209d7a1f3611
+Directory: 11.1/php8.3/fpm-alpine3.20
 
 Tags: 11.0.9-php8.3-apache-bookworm, 11.0-php8.3-apache-bookworm, 11-php8.3-apache-bookworm, php8.3-apache-bookworm, 11.0.9-php8.3-apache, 11.0-php8.3-apache, 11-php8.3-apache, php8.3-apache, 11.0.9-php8.3, 11.0-php8.3, 11-php8.3, php8.3, 11.0.9-apache-bookworm, 11.0-apache-bookworm, 11-apache-bookworm, apache-bookworm, 11.0.9-apache, 11.0-apache, 11-apache, apache, 11.0.9, 11.0, 11, latest
```

Comment:
`latest` and `11` should move to `11.1.x` now, right?

Reply:
https://github.com/docker-library/drupal/pull/277

Reply:
AH man, now I have to do the same for 10.4

Reply:
https://github.com/docker-library/drupal/pull/278